dist/jowl.spec

```diff
@@ -0,0 +1,87 @@
+%global npm_name jowl
+
+Name:           %{npm_name}
+Version:        2.0.2
+Release:        1%{?dist}
+Summary:        CLI for JSON operations with Lodash
+
+License:        MIT
+URL:            https://jowl.app
+Source0:        https://github.com/daxelrod/%{npm_name}/archive/refs/tags/v%{version}.tar.gz
+# Also at       https://registry.npmjs.org/{npm_name}/-/{npm_name}-{version}.tgz
+# however, npmjs.org has a build product which does not contain docs or tests
+Source1:        %{npm_name}-%{version}-nm-prod.tgz
+Source2:        %{npm_name}-%{version}-nm-dev.tgz
+Source3:        %{npm_name}-%{version}-bundled-licenses.txt
+
+BuildArch:      noarch
+ExclusiveArch:  %{nodejs_arches} noarch
+Requires:       nodejs
+BuildRequires:  nodejs-devel
+BuildRequires:  yarnpkg
+
+%description
+Jowl is a command-line filter for JSON expressions that uses plain JavaScript
+with Lodash. It takes JSON on standard in, and writes pretty-printed JSON to
+standard out.
+
+%prep
+%setup -q -n %{npm_name}-%{version}
+cp %{SOURCE3} .
+# Setup bundled runtime(prod) node modules
+tar xfz %{SOURCE1}
+mkdir -p node_modules
+pushd node_modules
+ln -s ../node_modules_prod/* .
+ln -s ../node_modules_prod/.bin .
+popd
+
+%build
+#nothing to do
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pr package.json src/bin/jowl.js src/lib/ \
+    %{buildroot}%{nodejs_sitelib}/%{npm_name}
+# Copy over bundled nodejs modules
+cp -pr node_modules node_modules_prod \
+    %{buildroot}%{nodejs_sitelib}/%{npm_name}
+
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}/bin
+install -p -D -m0755 src/bin/jowl.js %{buildroot}%{nodejs_sitelib}/%{npm_name}/bin/jowl
+mkdir -p %{buildroot}%{_bindir}
+ln -sf %{nodejs_sitelib}/%{npm_name}/bin/jowl %{buildroot}%{_bindir}/jowl
+
+# Fix the shebang because brp-mangle-shebangs fails to detect this properly (rhbz#1998924)
+# This is fixed in fc36 and above
+sed -e "s|^#!/usr/bin/env node$|#!/usr/bin/node|" \
+    -i %{buildroot}%{nodejs_sitelib}/%{npm_name}/bin/jowl \
```

Comment:
Isn't this a symlink?

Reply:
It's not, but to your point below, it probably should be.

Reply:
See below; making it a symlink breaks node's ability to find node_modules for reasons I don't understand. Going to leave it as-is, which is at least in line with how tape does it in Fedora.